### PR TITLE
RSDK-3979 - Remove "Get" from slam service method names

### PR DESCRIPTION
**Pull request by kkufieta**
Ticket: https://viam.atlassian.net/browse/RSDK-3979

Tested using fake slam.

This PR has to be merged before the corresponding PR in viam-cartographer can be merged: https://github.com/viamrobotics/viam-cartographer/pull/257

### CODE REVIEW

File: services/motion/builtin/builtin.go [hunk 1]
```diff
@@ -448,7 +448,7 @@ func (ms *builtIn) planMoveOnMap(
 	}
```
Comment (jeremyrhyde) [inline services/motion/builtin/builtin.go:448]: Should `GetPose` (line 414) also be changed?

File: services/navigation/builtin/builtin_test.go [hunk 2]
```diff
@@ -130,7 +130,7 @@ func TestStartWaypoint(t *testing.T) {
 	test.That(t, err, test.ShouldBeNil)
```
Comment (jeremyrhyde) [inline services/navigation/builtin/builtin_test.go:130]: Should `GetObstacles` be changed in navigation builtin.go?

File: services/slam/client_test.go [hunk 3]
```diff
@@ -119,26 +119,26 @@ func TestClientWorkingService(t *testing.T) {
 		workingSLAMClient, err := slam.NewClientFromConn(context.Background(), conn, "", slam.Named(nameSucc), logger)
 		test.That(t, err, test.ShouldBeNil)
 		// test get position
```
Comment (jeremyrhyde) [inline services/slam/client_test.go:121]: remove get from comments

File: services/slam/server_test.go [hunk 4]
```diff
@@ -270,7 +270,7 @@ func TestFailingServer(t *testing.T) {
 
 	t.Run("failing GetInternalState", func(t *testing.T) {
 		// GetInternalStateFunc error
```
Comment (jeremyrhyde) [inline services/slam/server_test.go:272]: fix comment

File: services/slam/server_test.go [hunk 5]
```diff
@@ -170,11 +170,11 @@ func TestWorkingServer(t *testing.T) {
...
-		injectSvc.GetPositionFunc = func(ctx context.Context) (spatial.Pose, string, error) {
+		injectSvc.PositionFunc = func(ctx context.Context) (spatial.Pose, string, error) {
 			return poseSucc, componentRefSucc, nil
 		}
 
-		injectSvc.GetPointCloudMapFunc = func(ctx context.Context) (func() ([]byte, error), error) {
+		injectSvc.PointCloudMapFunc = func(ctx context.Context) (func() ([]byte, error), error) {
 			reader := bytes.NewReader(pcd)
 			serverBuffer := make([]byte, chunkSizeServer)
 			f := func() ([]byte, error) {
```
Comment (jeremyrhyde) [inline services/slam/server_test.go:180]: Check comments on line 190 and 203

File: services/slam/slam.go [hunk 6]
```diff
@@ -57,10 +57,10 @@ func FromRobot(r robot.Robot, name string) (Service, error) {
 // Service describes the functions that are available to the service.
```
Comment (jeremyrhyde) [inline services/slam/slam.go:57]: [q] Do the collectors in the init need to be updated?

**Review by jeremyrhyde — APPROVED** [review]
LGTM, a few comments still need to be updated as well as the collector constants before merge.

File: services/slam/slam.go (hunk 6 above)
Comment (jeremyrhyde) [reply services/slam/slam.go:57]: image

Comment (kkufieta) [reply services/slam/slam.go:57]: yes, done

File: services/slam/server_test.go (hunk 5 above)
Comment (jeremyrhyde) [reply services/slam/server_test.go:180]: Yes

Comment (kkufieta) [reply services/slam/server_test.go:180]: I assumed that the comments are referring to the server functions, which are still `GetPosition` and `GetPointCloudMap`, and thus don't need to be changed.

Comment (jeremyrhyde) [reply services/slam/server_test.go:180]: Good point! Ignore my suggestion

**Pull request merged**